Userland/Libraries/LibC/time.cpp

```diff
@@ -205,7 +205,11 @@ char* asctime(const struct tm* tm)
 char* asctime_r(const struct tm* tm, char* buffer)
 {
     // Spec states buffer must be at least 26 bytes.
-    constexpr size_t assumed_len = 26;
+    // However, setting the length to 26 will crash for some tms because the year will be too big to fit.
+    // Additionally, asctime() already allocates for us 69 bytes, so there's no reason to not use them.
+    // Only 68 bytes are used to leave room for the null-terminator.
+
```

Comment:
Extraneous newline.

Reply:
In that case, what should asctime_r() do when strftime() tells it that the string didn't fit? Because we can't disobey the spec, asctime() has to return a char pointer, so the only other way to send the message that the method failed would be to return a null pointer, which would also crash. Should asctime_r() just return an incomplete string? These seem like spec issues, and I don't think I can make those decisions. I know that other time.h implementations that I've used don't crash, and return a string larger than 25 characters.